packages/grid/_modules_/grid/components/GridAutoSizer.tsx

```diff
@@ -70,6 +71,7 @@ export const GridAutoSizer = React.forwardRef<HTMLDivElement, AutoSizerProps>(fu
 
   const rootRef = React.useRef<HTMLDivElement | null>(null);
   const parentElement = React.useRef<HTMLElement | null>(null);
+  const apiRef = React.useContext(GridApiContext)!;
```

Comment:
This component is meant to be called `AutoSizer`, not `GridAutoSizer`, it's a generic helper, it can't contain API-related information.
```suggestion
```